test/test_models.py

```diff
@@ -313,3 +313,12 @@ def test_zoo_models(arch_name):
     model = models.__dict__[arch_name](pretrained=True)
     # Output checks
     assert isinstance(model, models.OCRPredictor)
+
+
+def test_resnet31():
+    resnet_model = models.Resnet31(input_size=(32, 128, 3))
+    resnet_input = tf.random.uniform(shape=[8, 32, 128, 3], minval=0, maxval=1)
+    resnet_out = resnet_model(inputs=resnet_input)
+    assert isinstance(resnet_out, tf.Tensor)
+    assert isinstance(resnet_model, tf.keras.Model)
+    assert resnet_out.numpy().shape == (8, 4, 32, 512)
```

Comment:
You can remove this, and only add a line right after L264
```
["resnet31", False, (32, 128, 3), (4, 32, 512)],
```
from what I understand